tests/console/ansible.pm

```diff
@@ -26,22 +26,29 @@ use transactional qw(trup_call check_reboot_changes);
 # sudo is used by ansible to become root
 # sudo has to be pop-out after installation otherwise
 # during the cleanup it will remove preinstalled salt packages
-my @pkgs = qw(sudo git-core);
+my @pkgs = qw(sudo git-core ansible);
 
 sub run {
     select_serial_terminal;
 
     # 1. System setup
 
-    if (is_sle('<15-SP7')) {
+    # poo#181136, 15-SP6 and above use from Systems Managent Module, 15-SP4 and 15-SP5 use LTSS
+    # PackageHub should not be covered
+    if (is_sle('<15-SP6') && !main_common::is_updates_tests()) {
```

Comment:
This will be always false as there is no sles less than sp7 tha is not in updates testing scope